src/gui/Clipboard.cpp

```diff
@@ -44,28 +44,26 @@ Clipboard::Clipboard(QObject* parent)
     connect(qApp, SIGNAL(aboutToQuit()), SLOT(clearCopiedText()));
 }
 
-void Clipboard::setText(const QString& text)
+void Clipboard::setText(const QString& text, bool clear)
 {
-    QClipboard* clipboard = QApplication::clipboard();
+    auto* clipboard = QApplication::clipboard();
 
-    QMimeData* mime = new QMimeData;
+    auto* mime = new QMimeData;
 #ifdef Q_OS_MACOS
     mime->setText(text);
     mime->setData("application/x-nspasteboard-concealed-type", text.toUtf8());
     clipboard->setMimeData(mime, QClipboard::Clipboard);
 #else
-    const QString secretStr = "secret";
-    QByteArray secretBa = secretStr.toUtf8();
     mime->setText(text);
-    mime->setData("x-kde-passwordManagerHint", secretBa);
+    mime->setData("x-kde-passwordManagerHint", QString("secret").toUtf8());
```

Comment:
This should be `QByteArrayLiteral("secret")`.

Reply:
done, I also added a nullptr check of clipboard in the setText function.